packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/RepeatableComponent/index.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Badge } from '@strapi/design-system/Badge';
+import { Box } from '@strapi/design-system/Box';
+import { Typography } from '@strapi/design-system/Typography';
+import { SimpleMenu, MenuItem } from '@strapi/design-system/SimpleMenu';
+import { stopPropagation } from '@strapi/helper-plugin';
+
+import CellValue from '../CellValue';
+
+function getMainFieldValue(field, name) {
+  return field[name] || field.id;
+}
+
+function getMainFieldType(field, name, defaultType) {
+  if (field[name]) {
```

Comment:
In which case could this be undefined?

Reply:
`getMainFieldType` is a leftover and can be removed.